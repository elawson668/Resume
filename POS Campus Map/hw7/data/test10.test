r
76
89
r
89
76
r
65
85
r
85
65
q
