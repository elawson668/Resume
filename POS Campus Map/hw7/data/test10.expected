First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to East Campus Athletic Village Arena:
	Walk NorthEast to (Folsom Library)
	Walk North to (Lally Hall)
	Walk NorthEast to (Intersection 134)
	Walk NorthEast to (Intersection 133)
	Walk NorthEast to (Troy Building)
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
	Walk East to (Quadrangle Complex)
	Walk East to (Intersection 108)
	Walk East to (Intersection 105)
	Walk East to (Admissions)
	Walk SouthEast to (Intersection 119)
	Walk NorthEast to (Intersection 118)
	Walk NorthEast to (Intersection 109)
	Walk East to (Burdett Avenue Residence Hall)
	Walk NorthEast to (Intersection 126)
	Walk East to (Intersection 128)
	Walk East to (Intersection 127)
	Walk North to (East Campus Athletic Village Arena)
Total distance: 1695.387 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from East Campus Athletic Village Arena to EMPAC:
	Walk South to (Intersection 127)
	Walk West to (Intersection 128)
	Walk West to (Intersection 126)
	Walk SouthWest to (Burdett Avenue Residence Hall)
	Walk West to (Intersection 109)
	Walk SouthWest to (Intersection 118)
	Walk SouthWest to (Intersection 119)
	Walk NorthWest to (Admissions)
	Walk West to (Intersection 105)
	Walk West to (Intersection 108)
	Walk West to (Quadrangle Complex)
	Walk West to (87 Gymnasium)
	Walk West to (Ricketts Building)
	Walk West to (Troy Building)
	Walk SouthWest to (Intersection 133)
	Walk SouthWest to (Intersection 134)
	Walk SouthWest to (Lally Hall)
	Walk South to (Folsom Library)
	Walk SouthWest to (EMPAC)
Total distance: 1695.387 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Patroon Manor to Blitman Residence Commons:
	Walk SouthWest to (Intersection 115)
	Walk South to (Rensselaer Apartment Housing Project RAHP B Site)
	Walk South to (Stacwyck Apartments)
	Walk SouthWest to (East Campus Athletic Village Arena)
	Walk South to (Intersection 127)
	Walk West to (Intersection 128)
	Walk West to (Intersection 126)
	Walk SouthWest to (Burdett Avenue Residence Hall)
	Walk West to (Intersection 109)
	Walk SouthWest to (Intersection 118)
	Walk SouthWest to (Intersection 119)
	Walk NorthWest to (Admissions)
	Walk West to (Intersection 105)
	Walk West to (Intersection 108)
	Walk West to (Intersection 132)
	Walk West to (Intersection 131)
	Walk West to (Intersection 130)
	Walk SouthWest to (Pittsburgh Building)
	Walk West to (Intersection 94)
	Walk SouthWest to (Winslow Building)
	Walk South to (Louis Rubin Memorial Approach)
	Walk West to (Blitman Residence Commons)
Total distance: 2511.445 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Blitman Residence Commons to Patroon Manor:
	Walk East to (Louis Rubin Memorial Approach)
	Walk North to (Winslow Building)
	Walk NorthEast to (Intersection 94)
	Walk East to (Pittsburgh Building)
	Walk NorthEast to (Intersection 130)
	Walk East to (Intersection 131)
	Walk East to (Intersection 132)
	Walk East to (Intersection 108)
	Walk East to (Intersection 105)
	Walk East to (Admissions)
	Walk SouthEast to (Intersection 119)
	Walk NorthEast to (Intersection 118)
	Walk NorthEast to (Intersection 109)
	Walk East to (Burdett Avenue Residence Hall)
	Walk NorthEast to (Intersection 126)
	Walk East to (Intersection 128)
	Walk East to (Intersection 127)
	Walk North to (East Campus Athletic Village Arena)
	Walk NorthEast to (Stacwyck Apartments)
	Walk North to (Rensselaer Apartment Housing Project RAHP B Site)
	Walk North to (Intersection 115)
	Walk NorthEast to (Patroon Manor)
Total distance: 2511.445 pixel units.